projects/js-packages/components/components/icon-tooltip/style.scss

```diff
@@ -91,6 +82,18 @@ $arrow-color: var( --jp-gray );
 .icon-tooltip-container {
 	// Recover events
 	pointer-events: all;
+
+	// __content styles here covers both the inline and portal modes
+	.components-popover__content {
+		padding: 24px;
+		width: 304px;
+		white-space: normal;
+		border-radius: 4px;
+		outline: none;
+		border: 1px solid $arrow-color;
+		max-width: 90vw;
+		margin: 0 10px;
```

Comment:
On smaller devices, the popover touches the edges, thus some margin along with max-width to fix that

| Before | After |
|--------|--------|
| ![IMG_5264](https://github.com/Automattic/jetpack/assets/18226415/e94d8909-5d6c-461b-8a6d-83406421dde1) | ![IMG_5265](https://github.com/Automattic/jetpack/assets/18226415/42bd8856-7f2b-4a85-87f1-db981afc5a07) |
| <img width="293" alt="Screenshot 2023-12-21 at 6 57 29 PM" src="https://github.com/Automattic/jetpack/assets/18226415/d00735ad-e3da-4a19-b30e-54cdfef93708"> | <img width="292" alt="Screenshot 2023-12-21 at 6 58 22 PM" src="https://github.com/Automattic/jetpack/assets/18226415/2142e7cc-19f1-40db-961e-0c5a8bafd647"> |